src/index.html

```diff
@@ -1,13 +1,54 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="cards" data-qa="card">
+      <img
+        class="cards__img"
+        src="images\imac.jpeg"
+        alt="imac_photo"
+      >
+
+      <h1 class="cards__text">
+        Моноблок APPLE A1419 iMac 27&quot Retina 5K (MNED2UA/A)
+      </h1>
+
+      <p class="cards__code">
+        Код товара: 195434
+      </p>
+
+      <div class="cards__reviewsTag">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="review">
+          Отзывов: 5
+        </p>
+      </div>
+
+      <div class="cards__priceTag">
+        <span class="priceText">Цена:</span>
```

Comment:
```suggestion
        <span class="card__price-text">Цена:</span>
```
We should not use two coding styles at the same time (in this case you additionally applied **camelCase**)